### ISSUE TITLE
GUI: UI elements redraw in odd locations, requiring client restart

### ISSUE THREAD

**Issue by InResponse** [magefree/mage#6804]
Version: 1.4.43V0
Issue type: bug
Severity: fatal
Steps to reproduce: unknown
Frequency: rare

This has now happened twice with 1.4.43V0. In the middle of a game, the UI froze temporarily. When user control returned, UI elements were shuffled around. (I don't know if it is relevant, but while the client was inaccessible, Task Manager listed System Interrupts at 100% CPU usage.)

![Annotation 2020-07-07 180143](https://user-images.githubusercontent.com/64223995/86854129-75526300-c07d-11ea-9418-e33d0d9489d0.png)
![Annotation 2020-07-07 182755](https://user-images.githubusercontent.com/64223995/86855278-a3d13d80-c07f-11ea-943f-447800e295e0.png)
As you can see, it looks as though part of the screen is updating, but with the incorrect origin point. Mouse movement caused the GUI to redraw at the cursor. Resizing, minimizing, and maximizing the window did not change this behavior.

There was no indication of an error or odd behavior in the client console until closing the client window. Then there was a single error, as shown below.

```
FATAL 2020-07-07 18:01:09,521 Card object on the cards panel was not removed  =>[AWT-EventQueue-0] Cards.loadCards 
FATAL 2020-07-07 18:01:09,522 MageCard name:null toolTiptext:null             =>[AWT-EventQueue-0] Cards.loadCards 
FATAL 2020-07-07 18:02:22,136 Card object on the cards panel was not removed  =>[AWT-EventQueue-0] Cards.loadCards 
FATAL 2020-07-07 18:02:22,136 MageCard name:null toolTiptext:null             =>[AWT-EventQueue-0] Cards.loadCards 
FATAL 2020-07-07 18:02:44,895 Card object on the cards panel was not removed  =>[AWT-EventQueue-0] Cards.loadCards 
FATAL 2020-07-07 18:02:44,895 MageCard name:null toolTiptext:null             =>[AWT-EventQueue-0] Cards.loadCards 
INFO  2020-07-07 18:02:45,957 Disconnecting...                                =>[AWT-EventQueue-0] SessionImpl.disconnect 
INFO  2020-07-07 18:02:45,980 Disconnecting DONE                              =>[AWT-EventQueue-0] SessionImpl.disconnect 
INFO  2020-07-07 18:02:45,980 Disconnected from remote task                   =>[AWT-EventQueue-0] MageFrame.disconnected 
FATAL 2020-07-07 18:02:46,344 MAGE Client UI error                            =>[Thread-532] EDTExceptionHandler.handle 
java.lang.IncompatibleClassChangeError
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=384m; support was removed in 8.0

```

**Comment by JayDi85** [comment]
That's error logs related to not clean up stack panel (see #213). BUT little freezes related to memory/disk issues (if you use HDD with antivirus then images load for new battlefield cards can take some time -- try to add images folder to antivirus ignore list).

Open launcher and try to add additional command for java -> client:
* `-Dsun.java2d.d3d=false` ([more details here](https://github.com/magefree/mage/issues/6029#issuecomment-564299739))

**Comment by JayDi85** [comment]
Related error with "freezed" cards in hand/stack were fixed in #7297.
